### PR TITLE
Protect sshd with 2FA

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/6088

### CODE REVIEW

File: ui/public/i18n/language.json [hunk 1]
```diff
@@ -841,6 +841,7 @@
     "validate_the_QRcode_token": "Verification code",
     "test_token": "Check code",
     "OtpCockpit_status": "Enable 2FA for the Server Manager",
+    "OtpSshd_status": "Enable 2FA for SSH (password only)",
```
Comment (gsanchietti) [inline ui/public/i18n/language.json]:
```suggestion
    "OtpSshd_status": "Enable 2FA for SSH",
```

File: ui/src/components/system/Settings.vue [hunk 2]
```diff
@@ -243,6 +243,29 @@
...
           </div>
         </div>
+        <div
+          v-if="view.otpIsLoaded && otp.OtpStatus"
+          :class="['form-group', errors.OtpSshd.hasError ? 'has-error' : '']"
+        >
+          <label
+            class="col-sm-2 control-label"
+            for="textInput-modal-markup"
+          >{{$t('settings.OtpSshd_status')}}</label>
```
Comment (gsanchietti) [inline ui/src/components/system/Settings.vue]: Can we add an inline-doc here?
The tool tip should state something like:
```
2FA is available only for password authentication.
Authentication with public keys will not be affected by 2FA.
```

Comment (stephdl) [reply ui/src/components/system/Settings.vue]: @gsanchietti done

File: ui/public/i18n/language.json (hunk 1 above)
Comment (stephdl) [reply ui/public/i18n/language.json]: done

**Pull request merged**